wls-wahlvorbereitung-service/src/test/java/de/muenchen/oss/wahllokalsystem/wahlvorbereitungservice/configuration/nfcconverter/NfcHelperTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c): it@M - Dienstleister für Informations- und Telekommunikationstechnik
+ * der Landeshauptstadt München, 2024
+ */
+package de.muenchen.oss.wahllokalsystem.wahlvorbereitungservice.configuration.nfcconverter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+import jakarta.servlet.http.Cookie;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class NfcHelperTest {
+
+    private static final String FIRST_NFD = "\u017f\u0307";
+
+    private static final String FIRST_NFC = "\u1e9b";
+
+    private static final String SECOND_NFD = "\u006f\u0302";
+
+    private static final String SECOND_NFC = "\u00f4";
+
+    private static final String THIRD_NFD = "\u0073\u0323\u0307";
+
+    private static final String THIRD_NFC = "\u1e69";
+
+    private static final String[] NFD_INPUT = new String[] { FIRST_NFD, SECOND_NFD, THIRD_NFD };
+
+    private static final String[] NFC_OUTPUT_EXPECTED = new String[] { FIRST_NFC, SECOND_NFC, THIRD_NFC };
+
+    @Test
+    void nfcConverterString() {
```

Comment:
Bei dem Test könntest du noch die Anpassungen die durch dich inspiriert wurden ergänzen: https://github.com/it-at-m/Wahllokalsystem/blob/dev/wls-briefwahl-service/src/test/java/de/muenchen/oss/wahllokalsystem/briefwahlservice/configuration/nfcconverter/NfcHelperTest.java